projects/hslayers/src/components/layermanager/layermanager-metadata.service.ts

```diff
@@ -147,7 +156,12 @@ export class HsLayerManagerMetadataService {
     }
   }
 
-  parseLayerInfo(layer: Layer, layerName: string, caps: any): void {
+  parseLayerInfo(
+    olLayer: Layer,
+    layerName: string,
+    caps: WMSGetCapabilitiesResponse,
+    fromSublayerParam: boolean
```

Comment:
I think fromSublayerParam param was deemed redundant and removed in my PR

Reply:
Yes, it seems so. Thanks!